src/components/Display.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+import { Textfit } from 'react-textfit';
+// import '../App.css';
+
+const Display = ({ value }) => (
+  <Textfit className="display" mode="single" max={70}>
+    {value}
```

Comment:
- [ ] its result prop should be a String with default value = 0 (Right now, it is only being initialized with the value 0 but 0 is not set as the default value, you can do so by equaling value to 0 at the function parameter)  ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))